### PR TITLE
i18n Occitan

### PR DESCRIPTION
**Pull request by Quenty31**
I copied pasted the .po file from weblate hoping it works this way.

### CODE REVIEW

File: pytition/locale/oc/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1,1214 +1,1223 @@
...
-"Project-Id-Version: \n"
+"Project-Id-Version: Occitan (Pytition)\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-10-26 17:56+0200\n"
-"PO-Revision-Date: 2019-10-26 18:00+0200\n"
+"POT-Creation-Date: 2019-11-04 19:36+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Quentin PAGÈS\n"
+"Language-Team: Occitan"
+">\n"
```
Comment (fallen) [inline pytition/locale/oc/LC_MESSAGES/django.po]: This line (">\n") looks strange, does it come from a bad copy paste?

**Comment by fallen** [comment]
The up to date file from which to translate is this one: https://github.com/pytition/Pytition/blob/master/pytition/locale/oc/LC_MESSAGES/django.po
Is this the one you took?

**Comment by Quenty31** [comment]
J'ai rien pris :)

**Comment by fallen** [comment]
Maybe you can have a look with @ButterFlyOfFire why the PO file on github and the one you sent in the Pull Request are not synchronised?

**Comment by BoFFire** [comment]
Hi, I'll check that.

**Comment by fallen** [comment]
@Quenty31 Do you want to merge this Pull Request? Or to wait for the translations to be 100% complete before merging?

**Comment by Quenty31** [comment]
Oui, si ça fonctionne je suis d’accord pour que ça soit déjà intégré :)

**Comment by fallen** [comment]
Alright, looks good to me!
Merging...
Thanks a lot :)

**Comment by Quenty31** [comment]
Many thanks! Looking forward to see it in action!
Still working on the rest of the translation.

**Comment by fallen** [comment]
It is now possible to see this translation online on the demo website: https://pytitiondemo.sionneau.net/
Thanks a lot!

**Comment by Quenty31** [comment]
79% now ;) I'll open another PR, I have to tune it a bit ^^

**Comment by BoFFire** [comment]
@Quenty31 Thanks :)

**Pull request merged**